docker-compose.yml

```diff
@@ -10,8 +10,10 @@ services:
       - maestro_traefik
     volumes:
       - ./data:/index/
+      - ./pylucene/:/python-docker
     environment:
       - SE_ALLOW_UNSAFE_SSL=TRUE
+      - DEBUG=TRUE
```

Comment:
The Dockerfile should not come with DEBUG=TRUE

@GWNLekkah should we also remove the SE_ALLOW_UNSAFE_SSL=TRUE?

Reply:
Should we create a docker-compose for developers as well?

Reply:
Yeah I like this idea

Reply:
I wouldn't remove the SE_ALLOW_UNSAFE_SSL, as it can be useful for testing the setup with self-signed certificates. I do like the idea of a dev docker setup as well. SSL setup is part of Traefik mostly, but I can change the docker files in a way that only the Traefik docker-compose file needs to be updated for enabling/disabling SSL.